variables.tf

```diff
@@ -57,6 +57,12 @@ variable "agents_min_count" {
   description = "Minimum number of nodes in a pool"
 }
 
+variable "max_surge" {
```

Comment:
Another thought, since this variable is used for default node pool, and other variables for this default node pool all have "agents_pool_" as name's prefix, could we rename this variable to `agents_pool_max_surge`?

Reply:
Sure, I totally agree.